views/timeline/simple.hbs

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link title="timeline-styles" rel="stylesheet" href="../../static/css/timeline-simple.css">
+    <script src="../../static/js/simpleTimeline.js"></script>
+
+</head>
+
+<body>
+    <h1>Top stories over the last 24 hours</h1>
+    <div class="flex-parent">
+        <div class="input-flex-container">
+            {{#each data}}
+            {{#if firstArticle}}
```

Comment:
If the two outputs of the if are so similar could it be simplified to:
<div class="input {{#if firstArticle}}active{{/if}}">
or is this too messy?